### PR TITLE
Feat/scout 262 more liked filters

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description
[SCOUT-262](https://playmakerpro.atlassian.net/browse/SCOUT-262?atlOrigin=eyJpIjoiYjExNmI1NjUwNzljNGMyZDljNDg4ODgzZjU2ZjZjMmIiLCJwIjoiaiJ9)

- added teams list params
- added liked teams& players filters on `notes` `reports`

### CODE REVIEW

File: src/pages/notes/index.tsx [hunk 1]
```diff
@@ -55,6 +57,9 @@ const NotesPage = () => {
     useState(false)
   const [noteToDeleteData, setNoteToDeleteData] = useState<INoteToDeleteData>()
 
+  const [filterLikedTeams, setLikedTeams] = useState(false)
+  const [filterLikedPlayers, setLikedPlayers] = useState(false)
```
Comment (jakubfiglak) [inline src/pages/notes/index.tsx:61]: Why did you create separate pieces of state for this? I don't see a reason why we couldn't just use `filters.onlyLikedPlayers` and `filters.onlyLikedTeams` directly to achieve the same result. There's a rule in state management that you shouldn't keep in state anything you can derive from some other state values - I think it's perfectly applicable here. In fact, you're not even deriving it, you want it to be exactly the same value, am I right?

File: src/pages/reports/index.tsx [hunk 2]
```diff
@@ -61,6 +63,9 @@ const ReportsPage = () => {
   const [reportToDeleteData, setReportToDeleteData] =
     useState<IReportToDeleteData>()
 
+  const [filterLikedTeams, setLikedTeams] = useState(false)
+  const [filterLikedPlayers, setLikedPlayers] = useState(false)
```
Comment (jakubfiglak) [inline src/pages/reports/index.tsx:67]: The same as in notes, I believe these `useState` calls are redundant

**Review by jakubfiglak — CHANGES_REQUESTED** [review]
Good job, just one thing to improve

File: src/pages/notes/index.tsx (hunk 1 above)
Comment (dejwi) [reply src/pages/notes/index.tsx]: yes you're right

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**